core/nginx/Dockerfile

```diff
@@ -17,18 +17,18 @@ ARG VERSION
 LABEL version=$VERSION
 
 RUN set -euxo pipefail \
-  ; apk add --no-cache certbot nginx nginx-mod-http-brotli nginx-mod-stream nginx-mod-mail openssl \
-  ; rm /etc/nginx/conf.d/stream.conf
+  ; apk add --no-cache certbot nginx nginx-mod-http-brotli nginx-mod-mail openssl dovecot-lua dovecot-pigeonhole-plugin
 
 COPY conf/ /conf/
 COPY --from=static /static/ /static/
 COPY *.py /
+COPY proxy.conf login.lua /
```

Comment:
Since these files are dovecot specific they should be stored in such a directory. 
In the github project, please store these files in a subfolder that makes it clear it is for dovecot.
Maybe the folder `core/nginx/dovecot`

In the image the files can be placed in `/dovecot_conf`

This makes it clear for other developers that these files are used for the dovecot proxy and not for nginx